semantic-conventions/src/tests/data/jinja/metrics/expected_no_whitespace.java

```diff
@@ -0,0 +1,19 @@
+package io.opentelemetry.instrumentation.api.metric;
+
+class Units {
+    /**
+    * Use this unit for Metric Instruments recording values
+    * representing fraction of a total.
+    **/
+    public static final String PERCENT = "%";
+    /**
+    * Use this unit for Metric Instruments recording values
+    * representing time.
+    **/
+    public static final String NANOSECOND = "NS";
+    /**
+    * Use this unit for Metric Instruments recording values
+    * representing connections.
+    **/
+    public static final String CONNECTIONS = "{connections}";
+}
```

Comment:
Is the final newline at EOF also trimmed? Git considers that bad style 😃

Reply:
Yeah it seems so. I managed to keep the newline at EOF by adding another newline 🤷. The existing `expected.java` is also missing the newline at the end. 

To not mess with the existing text/files, I added a new expected/template explicitly to simulate the `trim-whitespace` flag enabled.